Automattic/wp-calypso
# Update G Suite designs

This updates the G Suite screens with the designs from p6qnuF-4HE-p2

<img width="424" alt="screen shot 2017-05-25 at 17 26 09" src="https://cloud.githubusercontent.com/assets/275961/26459730/5c1419c8-416f-11e7-9025-b2e5ddbbc350.png">
<img width="750" alt="screen shot 2017-05-25 at 17 25 37" src="https://cloud.githubusercontent.com/assets/275961/26459731/5c183f62-416f-11e7-87a6-128da880f521.png">
<img width="428" alt="screen shot 2017-05-25 at 17 25 14" src="https://cloud.githubusercontent.com/assets/275961/26459732/5c18c568-416f-11e7-8d0c-4af0b1680380.png">
<img width="1065" alt="screen shot 2017-05-25 at 17 24 55" src="https://cloud.githubusercontent.com/assets/275961/26459733/5c21364e-416f-11e7-97dc-9cd1b23f6f8c.png">
  
#### Testing instructions
  
1. Run `git checkout update/g-suite` and start your server
2. Open the [`Email page` page](http://calypso.localhost:3000/domains/manage/[domain]/email/[domain])
3. Check that the designs look as above
2. Go through the domain purchase flow (http://calypso.localhost:3000/domains/add/[domain]/google-apps/[domain])
3. Check that the designs look as above

#### Reviews
